python/pip_install/pip_repository.bzl

```
@@ -178,6 +183,11 @@ pip_repository_attrs = {
         default = False,
         doc = "Create the repository in incremental mode.",
     ),
+    "pip_platform_definitions": attr.label_keyed_string_dict(
+        doc = """
+A map of select keys to platform definitions in the form <platform>-<python_version>-<implementation>-<abi>"
```

```suggestion
A map of select keys to Python platform definitions in the form <platform>-<python_version>-<implementation>-<abi>"
```

Just a nit, but helpful to distinguish from a Bazel `platform`.